src/NServiceBus.AcceptanceTests/Core/Recoverability/When_custom_policy_executed.cs

```diff
@@ -15,6 +15,8 @@ public class When_custom_policy_executed : NServiceBusAcceptanceTest
         [Test]
         public async Task Should_provide_error_context_to_policy()
         {
+            Requires.DelayedDelivery();
```

Comment:
Nitpick: We should be able to make this one not need delayed delivery as well

Reply:
The current policy here seems to be verifying that the policy is invoked multiple times. Is that actually something you'd want to change just to remove the delayed delivery requirement?